### ISSUE TITLE
Path issue when using cargo command in remote buffer

### ISSUE THREAD

**Issue by sunng87** [ayrat555/cargo-mode#14]
Thank you for building this mode!

I'm using this cargo mode from my remote buffer. It seems it always finds `cargo` command from `/sbin/cargo` instead of my `PATH`. 

```
/sbin/cargo clippy
/bin/sh: 2: /sbin/cargo: not found

Comint exited abnormally with code 127 at Thu Sep 26 15:27:26, duration 0.60 s

```

I used to have similar issue with eglot finding `rust-analyzer` but when add `(add-to-list 'tramp-remote-path 'tramp-own-remote-path)` it becomes fine. I think cargo-mode should also respect this path settings from tramp when calling cargo binary.

**Comment by ayrat555** [comment]
@sunng87 what do you think about setting `cargo-path-to-bin` var manually?

**Comment by sunng87** [comment]
We can use var as fallback. But it's still nice if we can follow the PATH

**Comment by sunng87** [comment]
This should be fixed in #15.
